packages/components/datepicker/README.mdx

```diff
@@ -1,27 +1,19 @@
 ---
 title: 'Datepicker'
-status: 'beta'
+status: 'alpha'
```

Comment:
Out of curiosity: Why this component is in alpha? Do we use `alpha`? I thought that we only use `beta` and then we move component to public release.

Reply:
This is because we had designs for the Alpha note but not for the Beta one, but we could use both actually.
But would need to have a copy for the Beta version note to display on the page

Reply:
I couldn't find any guidelines or docs in the repo for releases/deployments. We will need to work on our release processes, and document it. for now we will stay with what we have `alpha` and as it's also make sense in [Software Release Life Cycle](https://en.wikipedia.org/wiki/Software_release_life_cycle)

Reply:
Both `alpha` and `beta` are aligned with [Software Release Life Cycle](https://en.wikipedia.org/wiki/Software_release_life_cycle), I don't mind really, I just think we should be consistent. We didn't use alpha for v4, this is where my questions is coming from. (We actually didn’t use it for `Pagination` neither as far as I remember? 🤔 ).